queries/gularen/highlights.scm

```diff
@@ -0,0 +1,88 @@
+(chapter) @markup.heading.1
+
+(section) @markup.heading.2
+
+(subsection) @markup.heading.3
+
+(subtitle) @markup.heading.4
+
+(bullet) @markup.list
+
+(index) @number
+
+(v) @markup.list.checked
+
+(x) @markup.list.unchecked
+
+(bold) @markup.strong
+
+(italic) @markup.italic
+
+(monospaced) @markup.raw
+
+[
+  (line_break)
+  (page_break)
+  (dinkus)
+] @punctuation.special
+
+(code_inline) @markup.raw
+
+(code_block
+  (code_content) @markup.raw.block)
+
+(code_lang) @label
```

Comment:
Is this really a top-level node? Shouldn't this be
```
(code_block
  (code_lang) @label
  (code_content) @markup.raw.block)
```

Reply:
And what about `code_inline_lang`?

Reply:
`code_block` does not have `code_lang` children, only `code_block_lang` does

Reply:
for `code_inline_lang` I removed it in my latest commit, its very hard to implement it on `tree-sitter`